contrib/golang/filters/http/test/golang_integration_test.cc

```diff
@@ -76,7 +75,7 @@ TEST_P(GolangIntegrationTest, Echo) {
   EXPECT_EQ("403", response->headers().getStatusValue());
 
   // check body for echo
-  auto body = StringUtil::toUpper(absl::StrFormat("forbidden from go, path: %s\r\n", path));
+  auto body = StringUtil::toUpper(absl::StrFormat("%s, path: %s\r\n", echo_body, path));
   EXPECT_EQ(body, StringUtil::toUpper(response->body()));
```

Comment:
`StringUtil::toUpper` is not necessary.

Reply:
Yes, it could delete now. Previously, headers were case insensitive.